docs/reference-architectures/sap/index.md

```diff
@@ -0,0 +1,418 @@
+    focuses on SAP HANA. The SAP HANA in-memory data platform provides high availability at
+    the data tier through the HANA System Replication (HSR) feature, which
+    implements a manual failover.
+ 
+
+## Recommendations
+Your requirements might differ from the architecture described here. Use these recommendations as a starting point.
+
+### SAP application on Azure
+The common practice for achieving high availability is through redundancy of
+critical components. In this distributed installation of the SAP application on
+a centralized database, the base installation is replicated to achieve high
+availability. For each layer of the architecture, the high availability design
+varies depending on the components as follows:
+
+-	**ASCS.** For high availability of ASCS on Azure Windows virtual machines,
+    WSFC is used with SIOS DataKeeper to implement the cluster shared volume.
+    For implementation details, see [Clustering SAP ASCS on Azure][clustering].
+
+- **Application servers.** High availability is achieved by load balancing
+    traffic within a pool of application servers.
+
+-	**Database tier.** For SAP HANA running on Azure, the high availability approach is to implement HSR. 
+    This replication solution doesn’t offer automatic failover. To do that, use additional HA extensions from the Linux distribution.
+
+### Availability sets
+Putting virtual machines that perform the same role into an availability sets
+guards against downtime caused by Azure infrastructure maintenance and
+contributes to meeting [service level agreements][sla] (SLAs). Availability sets help minimize the impact of resource capacity during
+maintenance. Three or more virtual machines per availability set is recommended.
+
+Remember not to mix servers of different roles in the same availability set. For
+example, don’t place a ASCS node in the same availability set with the
+application server. When an availability set contains only virtual machines
+performing the same role, the Azure fabric tries to ensure that the services
+provided by the servers in the set remain available even during system failure.
+
+### Subnets
+Using multiple subnets, you can control different network security profiles in a
+consistent manner. For example, end users connect to SAP application servers
+during their normal business operations, but only administrators need access to
+the database servers. Place application servers on a separate subnet so you can
+secure them more easily by managing the subnet security policies, not the
+individual servers.
+
+### Load balancers
+[SAP Web Dispatcher][sap-dispatcher]. handles load balancing of HTTP(S) traffic to
+ dual-stack servers (ABAP and Java). SAP has advocated single-stack application servers for years, so very few
+applications run on a dual-stack deployment model nowadays. The Azure Load Balancer 
+depicted in the architecture diagram implements the high availability cluster for the SAP Web Dispatcher.
```

Comment:
As I understand, it's not really that the LB *implements* the cluster, it's that the LB enables the failover to happen.